### ISSUE TITLE
[BUG] RenderForm Conditional Formatting Not Rendering Components Properly

### ISSUE THREAD

**Issue by crivera305** [formio/formio.js#3055]
## Environment
Please provide as many details as you can:

- Hosting type
  - [] Form.io
  - [x] Local deployment
    - Version: 
- Formio.js version: 4.9.11 - Latest
- Frontend framework: Bootstrap
- Browser: Chrome / Safari / Firefox
- Browser version: All

## Steps to Reproduce

1. Goto Link to: https://jsfiddle.net/0mkns9av/
2. Notice Behavior Issues Component loads after some time empty; its missing its internal components - (FAIL)

** if you check the form code you can see that Behavior Issues Panel Component contains more components within it.

## Expected behavior
When some one saves a form answer with "Yes" for behavior issues radio button: 
- the Behavior Issues Panel Component should show
- the components within the Behavior Issues Panel Component should show also

## Observed behavior
When we load the form answer
- "Yes" is selected to show Behavior Issues radio
- Behavior Issues panel component takes considerable amount of time to load on the form ( ??? )
- Behavior Issues panel component is displayed but is empty ( FAIL ) 

there are other components defined to show up within this panel yet it loads blank every time.

Trying to come up with a work around. I tried to:
- click "No" on Behavior Issues radio
- then click "Yes" on Behavior Issues radio
- Behavior Issues panel component loads properly with the components inside of it
- but all the previous saved values are gone ( FAIL )

## Example
https://jsfiddle.net/0mkns9av/

For code or form JSON, please enclose in a code block:

```js
// FORM CODE
{"components":[{"label":"z","tableView":false,"key":"z","type":"well","input":false,"components":[{"html":"<h4 style=\"text-align:center;\"><strong>Shift Report</strong></h4>","label":"Content","refreshOnChange":false,"tableView":false,"key":"content","type":"content","input":false}]},{"label":"Columns","columns":[{"components":[{"label":"Date","useLocaleSettings":true,"format":"MM-dd-yyyy","tableView":false,"enableMinDateInput":false,"datePicker":{"disableWeekends":false,"disableWeekdays":false},"enableMaxDateInput":false,"enableTime":false,"calculateServer":false,"key":"date","type":"datetime","input":true,"suffix":"<i ref=\"icon\" class=\"fa fa-calendar\" style=\"\"></i>","widget":{"type":"calendar","displayInTimezone":"viewer","language":"en","useLocaleSettings":true,"allowInput":true,"mode":"single","enableTime":false,"noCalendar":false,"format":"MM-dd-yyyy","hourIncrement":1,"minuteIncrement":1,"time_24hr":false,"minDate":null,"disableWeekends":false,"disableWeekdays":false,"maxDate":null},"hideOnChildrenHidden":false},{"label":"Address/Building Name:","description":"If this is for Housing - please make sure you are on the correct dropdown","tooltip":"Indicate the name of the building. Example: Treatment office, Detox facility, 123 Main Street, Housing, etc.","spellcheck":true,"tableView":true,"calculateServer":false,"validate":{"required":true},"type":"textfield","input":true,"key":"textField1","hideOnChildrenHidden":false},{"label":"Select Shift:","tableView":true,"data":{"values":[{"label":"1st","value":"1st"},{"label":"2nd","value":"2nd"},{"label":"3rd","value":"3rd"}]},"selectThreshold":0.3,"calculateServer":false,"validate":{"required":true},"type":"select","indexeddb":{"filter":{}},"input":true,"key":"select1","hideOnChildrenHidden":false}],"width":6,"offset":0,"push":0,"pull":0,"size":"md"},{"components":[{"label":"Who else was on the shift with you? ","spellcheck":true,"tableView":true,"multiple":true,"calculateServer":false,"validate":{"multiple":true},"type":"textfield","input":true,"key":"textField3","hideOnChildrenHidden":false}],"width":6,"offset":0,"push":0,"pull":0,"size":"md"}],"tableView":false,"key":"columns2","type":"columns","input":false},{"label":"Columns","columns":[{"components":[{"label":"Incidents","tableView":false,"key":"incidents","type":"well","input":false,"components":[{"label":"Incident(s) on shift","optionsLabelPosition":"right","description":"Did an incident occur on your shift?","inline":false,"tableView":false,"values":[{"label":"Yes","value":"yes","shortcut":""},{"label":"No","value":"no","shortcut":""}],"calculateServer":false,"key":"didAnIncidentOccurOnYourShift","type":"radio","input":true,"hideOnChildrenHidden":false},{"label":"Have you filed an incident report?","optionsLabelPosition":"right","inline":false,"tableView":false,"values":[{"label":"Yes","value":"yes","shortcut":""},{"label":"No","value":"no","shortcut":""}],"calculateServer":false,"key":"haveYouFiledAnIncidentReport","conditional":{"show":true,"when":"didAnIncidentOccurOnYourShift","eq":"yes"},"type":"radio","input":true,"hideOnChildrenHidden":false},{"html":"<p>All Incidents must be reported. If you believe an Incident occurred on your shift and it was not reported, please let your supervisor know immediately.&nbsp;</p><p>We track all incidents to ensure the safety of both staff and clients.&nbsp;</p><p><a href=\"/client/cform/51/new\"><strong>Click this link to fill out a new incident report.&nbsp;</strong></a></p>","label":"Content","refreshOnChange":false,"tableView":false,"key":"content1","conditional":{"show":true,"when":"haveYouFiledAnIncidentReport","eq":"no"},"type":"content","input":false,"hideOnChildrenHidden":false},{"label":"Please enter incident ID here. If there were multiple incidents, click +Add Another and enter it on that line.","placeholder":"Enter Incident ID number here","tooltip":"Incident ID can be found on the far left of the list of incident reports.","mask":false,"spellcheck":true,"tableView":false,"multiple":true,"defaultValue":[null],"delimiter":false,"requireDecimal":false,"inputFormat":"plain","calculateServer":false,"validate":{"multiple":true},"key":"pleaseEnterIncidentReportNumberH","conditional":{"show":true,"when":"haveYouFiledAnIncidentReport","eq":"yes"},"type":"number","input":true,"hideOnChildrenHidden":false}],"hideOnChildrenHidden":false}],"width":6,"offset":0,"push":0,"pull":0,"size":"md"},{"components":[{"label":"Issues","tableView":false,"key":"issues","type":"well","input":false,"components":[{"html":"<p><i>If any of these issues occurred, please describe in the appropriate boxes below.&nbsp;</i></p>","label":"Content","refreshOnChange":false,"tableView":false,"key":"content2","type":"content","input":false,"hideOnChildrenHidden":false},{"label":"Behavioral issues","optionsLabelPosition":"right","description":"Did any behavioral issues occur on your shift?","inline":true,"tableView":false,"values":[{"label":"Yes","value":"yes","shortcut":""},{"label":"No","value":"no","shortcut":""}],"calculateServer":false,"validate":{"required":true},"key":"didAnyBehavioralIssuesOccurOnYourShift","type":"radio","input":true,"hideOnChildrenHidden":false},{"label":"Medical/ Nursing issues ","optionsLabelPosition":"right","description":"Did any Medical/ Nursing issues occur on your shift?","inline":true,"tableView":false,"values":[{"label":"Yes","value":"yes","shortcut":""},{"label":"No","value":"no","shortcut":""}],"calculateServer":false,"validate":{"required":true},"key":"didAnMedicalIssuesOccurOnYourShift","type":"radio","input":true,"hideOnChildrenHidden":false},{"label":"Clinical issues","optionsLabelPosition":"right","description":"Did anything occur on your shift that needs to be addressed specifically by a Therapist?","inline":true,"tableView":false,"values":[{"label":"Yes","value":"yes","shortcut":""},{"label":"No","value":"no","shortcut":""}],"calculateServer":false,"validate":{"required":true},"key":"didAnyClinicalIssuesOccurOnYourShift","type":"radio","input":true,"hideOnChildrenHidden":false},{"label":"Are there any unresolved Maintenance Issues? ","optionsLabelPosition":"right","tooltip":"Any maintenance issues should be reported to your Supervisor or be reported via the Maintenance Request Form. ANY Maintenance issue that is dangerous or could effect Operations should be communicated immediately.","inline":true,"tableView":false,"values":[{"label":"Yes","value":"yes","shortcut":""},{"label":"No","value":"no","shortcut":""}],"calculateServer":false,"validate":{"required":true},"key":"didAnyMaintenanceissuesOccurOnYourShift","type":"radio","input":true}],"hideOnChildrenHidden":false}],"width":6,"offset":0,"push":0,"pull":0,"size":"md"}],"tableView":false,"key":"columns3","type":"columns","input":false},{"title":"Behavioral Issues:","theme":"info","collapsible":false,"tableView":false,"key":"behavioralIssues","conditional":{"show":true,"when":"didAnyBehavioralIssuesOccurOnYourShift","eq":"yes"},"type":"panel","label":"Behavioral Issues:","input":false,"components":[{"label":"Select the issues that occurred on your shift","optionsLabelPosition":"right","tableView":false,"defaultValue":{"":false,"clientsNotFollowingRules":false,"blockedAmaAta":false,"clientsActingRomantically":false,"clientsDisrespectfulToStaffOtherClients":false,"clientsNotParticipatingInTreatment":false,"contrabandFound":false,"clientsUnhappyWithTreatment":false,"otherPleaseDescribe":false,"clientLeftAmaAta":false,"medicationAbuse":false},"values":[{"label":"Clients not following rules","value":"clientsNotFollowingRules","shortcut":""},{"label":"Blocked AMA/ATA","value":"blockedAmaAta","shortcut":""},{"label":"Client left AMA/ATA","value":"clientLeftAmaAta","shortcut":""},{"label":"Clients disrespectful to staff/other clients","value":"clientsDisrespectfulToStaffOtherClients","shortcut":""},{"label":"Clients not participating in treatment","value":"clientsNotParticipatingInTreatment","shortcut":""},{"label":"Contraband found","value":"contrabandFound","shortcut":""},{"label":"Clients unhappy with treatment","value":"clientsUnhappyWithTreatment","shortcut":""},{"label":"Clients acting romantically","value":"clientsActingRomantically","shortcut":""},{"label":"Medication Abuse","value":"medicationAbuse","shortcut":""},{"label":"Other (please describe)","value":"otherPleaseDescribe","shortcut":""}],"calculateServer":false,"validate":{"required":true},"key":"selectTheIssuesThatOccurredOnYourShift","type":"selectboxes","input":true,"inputType":"checkbox"},{"label":"If other issue, please describe","spellcheck":true,"tableView":true,"calculateServer":false,"conditional":{"show":true,"when":"selectTheIssuesThatOccurredOnYourShift","eq":"otherPleaseDescribe"},"type":"textfield","input":true,"key":"textField14"},{"label":"Client Name/Behavioral Issue:(use one line per client/ issue. Click +Add Another for a new line)","placeholder":"Example: John D / was talking about leaving AMA","description":"Use each field to enter client/issue separately. If mulitple clients are involved with same the issue, use the same field to describe.","autoExpand":true,"spellcheck":true,"tableView":true,"multiple":true,"defaultValue":[""],"calculateServer":false,"validate":{"multiple":true},"type":"textarea","rows":2,"input":true,"key":"textArea2"}]},{"title":"Medical/Nursing Issues","theme":"warning","collapsible":false,"tableView":false,"key":"medicalNursingIssues","conditional":{"show":true,"when":"didAnMedicalIssuesOccurOnYourShift","eq":"yes"},"type":"panel","label":"Medical/Nursing Issues","input":false,"components":[{"label":"Medical/Nursing Issues to be addressed: ","optionsLabelPosition":"right","tableView":false,"defaultValue":{"":false,"medicationRefusalNotPreviouslyCommunicated":false,"issueWithMedicationGeneral":false,"clientSentToEr911Called":false,"clientPresentingSymptoms":false,"cravingsRelapsePotential":false,"other":false,"otherPleaseDescribe":false,"medicationAbuse":false},"values":[{"label":"Medication Refusal (not previously communicated)","value":"medicationRefusalNotPreviouslyCommunicated","shortcut":""},{"label":"Issue with Medication (general)","value":"issueWithMedicationGeneral","shortcut":""},{"label":"Client sent to ER/911 called","value":"clientSentToEr911Called","shortcut":""},{"label":"Client presenting symptoms ","value":"clientPresentingSymptoms","shortcut":""},{"label":"Cravings/Relapse Potential","value":"cravingsRelapsePotential","shortcut":""},{"label":"Medication Abuse","value":"medicationAbuse","shortcut":""},{"label":"Other (please describe)","value":"otherPleaseDescribe","shortcut":""}],"calculateServer":false,"validate":{"required":true},"key":"medicalNursingIssuesToBeAddressed","type":"selectboxes","input":true,"inputType":"checkbox"},{"label":"If other issue, please describe","spellcheck":true,"tableView":true,"calculateServer":false,"conditional":{"show":true,"when":"medicalNursingIssuesToBeAddressed","eq":"otherPleaseDescribe"},"type":"textfield","input":true,"key":"textField15"},{"label":"Client/ Medical or Nursing Issue:(use one line per client/ issue. Click +Add Another for a new line)","placeholder":"Example: Jane D / refused to take her meds, but eventually did","description":"Use each field to enter client/issue separately. If mulitple clients are involved with the same issue, use the same field to describe.","autoExpand":true,"spellcheck":true,"tableView":true,"multiple":true,"calculateServer":false,"validate":{"multiple":true},"type":"textarea","rows":2,"input":true,"key":"textArea3"}]},{"title":"Clinical Issues","theme":"success","collapsible":false,"tableView":false,"key":"clinicalIssues","conditional":{"show":true,"when":"didAnyClinicalIssuesOccurOnYourShift","eq":"yes"},"type":"panel","label":"Clinical Issues","input":false,"components":[{"label":"Clinical Issues to be addressed: ","optionsLabelPosition":"right","tableView":false,"defaultValue":{"":false,"aggressionViolence":false,"anxiety":false,"cravings":false,"griefGuilt":false,"familyRelationshipIssues":false,"mentalHealthIssues":false,"relapsePrevention":false,"selfCare":false,"otherPleaseDescribe":false},"values":[{"label":"Aggression/Violence","value":"aggressionViolence","shortcut":""},{"label":"Anxiety","value":"anxiety","shortcut":""},{"label":"Cravings","value":"cravings","shortcut":""},{"label":"Grief/Guilt","value":"griefGuilt","shortcut":""},{"label":"Family/Relationship Issues","value":"familyRelationshipIssues","shortcut":""},{"label":"Mental Health Issues","value":"mentalHealthIssues","shortcut":""},{"label":"Relapse Prevention","value":"relapsePrevention","shortcut":""},{"label":"Hygiene/Self-Care","value":"hygieneSelfCare","shortcut":""},{"label":"Other (please describe)","value":"otherPleaseDescribe","shortcut":""}],"calculateServer":false,"validate":{"required":true},"key":"clinicalIssuesToBeAddressed","type":"selectboxes","input":true,"inputType":"checkbox"},{"label":"If other issue, please describe","spellcheck":true,"tableView":true,"calculateServer":false,"conditional":{"show":true,"when":"clinicalIssuesToBeAddressed","eq":"otherPleaseDescribe"},"type":"textfield","input":true,"key":"textField16"},{"label":"Client/ Clinical Issue:(use one line per client/ issue. Click +Add Another for a new line)","placeholder":"Example: Bill D / was isolating and depressed","description":"Use each field to enter client/issue separately. If mulitple clients are involved with the same issue, use the same field to describe.","autoExpand":true,"spellcheck":true,"tableView":true,"multiple":true,"calculateServer":false,"validate":{"multiple":true},"type":"textarea","rows":2,"input":true,"key":"textArea4"}]},{"title":"Maintenance Issues: ","collapsible":false,"tableView":false,"key":"maintenanceIssues","conditional":{"show":true,"when":"didAnyMaintenanceissuesOccurOnYourShift","eq":"yes"},"type":"panel","label":"Maintenance Issues: ","input":false,"components":[{"label":"Classify the Maintenance Issue:","optionsLabelPosition":"right","tableView":false,"values":[{"label":"Air/Heating Unit","value":"airHeatingUnit","shortcut":""},{"label":"Broken items (describe)","value":"brokenItemsDescribe","shortcut":""},{"label":"Doors/Door Lock","value":"doorsDoorLock","shortcut":""},{"label":"Electrical Unit","value":"electricalUnit","shortcut":""},{"label":"General Maintenance/Cleanliness","value":"generalMaintenanceCleanliness","shortcut":""},{"label":"Plumbing/Water","value":"plumbingWater","shortcut":""},{"label":"Structural ","value":"structural","shortcut":""},{"label":"Other (please describe)","value":"otherPleaseDescribe","shortcut":""}],"calculateServer":false,"key":"classifyTheMaintenanceIssue","type":"selectboxes","input":true,"inputType":"checkbox","defaultValue":{"":false}},{"label":"If other Issue, please describe","spellcheck":true,"tableView":true,"calculateServer":false,"conditional":{"show":true,"when":"clinicalIssuesToBeAddressed","eq":"otherPleaseDescribe"},"type":"textfield","input":true,"key":"textField17"},{"label":"Maintenance issues:(use one line per issue Click +Add Another for a new line)","placeholder":"Example: light bulb in the kitchen broke","description":"Use each field to enter Maintenance Issue separately. If mulitple clients are involved with the same issue, use the same field to describe.","autoExpand":true,"spellcheck":true,"tableView":true,"multiple":true,"calculateServer":false,"validate":{"multiple":true},"type":"textarea","rows":2,"input":true,"key":"textArea5"}]},{"label":"Shift Tasks: ","optionsLabelPosition":"right","tableView":false,"defaultValue":{"":false,"roundsCompleted":false,"vehicleInspected":false,"all":false,"admissionsPaperworkComplete":false,"allUAsObtained":false,"uAsObtainedIfNotLetTheStaffOnTheNextShiftKnow":false},"values":[{"label":"Rounds Completed","value":"roundsCompleted","shortcut":""},{"label":"Vehicle Inspected","value":"vehicleInspected","shortcut":""},{"label":"Admissions Paperwork Complete","value":"admissionsPaperworkComplete","shortcut":""},{"label":"U/A's obtained. (If not let the staff on the next shift know)","value":"uAsObtainedIfNotLetTheStaffOnTheNextShiftKnow","shortcut":""}],"calculateServer":false,"key":"shiftTasks","type":"selectboxes","input":true,"inputType":"checkbox"},{"label":"Housekeeping: Rooms of discharged clients that need to be prepped for next client(use one line per room Click +Add Another for a new line)","placeholder":"Example: Unit 112/  Room B","spellcheck":true,"tableView":true,"multiple":true,"calculateServer":false,"validate":{"multiple":true},"type":"textfield","input":true,"key":"textField2"},{"label":"Columns","columns":[{"components":[{"label":"Number of admissions","placeholder":"enter number here","mask":false,"spellcheck":true,"tableView":false,"delimiter":false,"requireDecimal":false,"inputFormat":"plain","calculateServer":false,"key":"number","type":"number","input":true,"hideOnChildrenHidden":false},{"label":"# of Male Clients","placeholder":"enter number here","mask":false,"spellcheck":true,"tableView":false,"delimiter":false,"requireDecimal":false,"inputFormat":"plain","calculateServer":false,"validate":{"required":true},"key":"ofMaleClients","type":"number","input":true,"hideOnChildrenHidden":false},{"label":"# of AMAs/ATAs:","placeholder":"enter number here","spellcheck":true,"tableView":true,"calculateServer":false,"validate":{"required":true},"type":"textfield","input":true,"key":"textField5","hideOnChildrenHidden":false},{"label":"# of Detox Clients:","placeholder":"enter number here","spellcheck":true,"tableView":true,"calculateServer":false,"type":"textfield","input":true,"key":"textField6","hideOnChildrenHidden":false},{"label":"# of Suicide Precaution Clients","placeholder":"enter number here","mask":false,"spellcheck":true,"tableView":false,"delimiter":false,"requireDecimal":false,"inputFormat":"plain","calculateServer":false,"key":"numberOfSuicidePrecautionPatients","type":"number","input":true,"hideOnChildrenHidden":false},{"label":"Names of Clients currently on Suicide Precautions(One person per line, click +Add Another to add a line)","placeholder":"Enter First Name & Last Initial (one person per line)","spellcheck":true,"tableView":true,"multiple":true,"calculateServer":false,"validate":{"multiple":true},"type":"textfield","input":true,"key":"textField12","hideOnChildrenHidden":false}],"width":6,"offset":0,"push":0,"pull":0,"size":"md"},{"components":[{"label":"Number of discharges","placeholder":"enter number here","mask":false,"spellcheck":true,"tableView":false,"delimiter":false,"requireDecimal":false,"inputFormat":"plain","calculateServer":false,"key":"numberOfDischarges","type":"number","input":true,"hideOnChildrenHidden":false},{"label":"# of Female Clients","placeholder":"enter number here","mask":false,"spellcheck":true,"tableView":false,"delimiter":false,"requireDecimal":false,"inputFormat":"plain","calculateServer":false,"validate":{"required":true},"key":"ofMaleClients1","type":"number","input":true,"hideOnChildrenHidden":false},{"label":"# of MAT(Medication Assisted Therapy) Clients:","placeholder":"enter number here","mask":false,"spellcheck":true,"tableView":false,"delimiter":false,"requireDecimal":false,"inputFormat":"plain","calculateServer":false,"key":"number1","type":"number","input":true,"hideOnChildrenHidden":false},{"label":"Names of Clients currently on MAT Therapy(One person per line, click +Add Another to add a line)","placeholder":"Enter First Name & Last Initial (one person per line)","spellcheck":true,"tableView":true,"multiple":true,"calculateServer":false,"validate":{"multiple":true},"type":"textfield","input":true,"key":"textField13","hideOnChildrenHidden":false}],"width":6,"offset":0,"push":0,"pull":0,"size":"md"}],"tableView":false,"key":"columns","type":"columns","input":false},{"label":"Shift Notes","autoExpand":true,"spellcheck":true,"tableView":true,"calculateServer":false,"type":"textarea","rows":2,"input":true,"key":"textArea1"},{"type":"button","label":"Submit","key":"submit","disableOnInvalid":true,"input":true,"tableView":false}]}
```

```js
// ANSWER CODE
{"date": "2020-07-16T00:00:00-04:00", "select1": "2nd", "textArea1": "", "textArea2": ["test"], "shiftTasks": {"": false, "all": false, "allUAsObtained": false, "roundsCompleted": false, "vehicleInspected": false, "admissionsPaperworkComplete": false, "uAsObtainedIfNotLetTheStaffOnTheNextShiftKnow": false}, "textField1": "test", "textField2": [""], "textField3": ["1", "2", "3"], "textField5": "", "textField6": "", "textField12": [""], "textField13": [""], "didAnIncidentOccurOnYourShift": "no", "didAnMedicalIssuesOccurOnYourShift": "", "didAnyClinicalIssuesOccurOnYourShift": "", "didAnyBehavioralIssuesOccurOnYourShift": "yes", "selectTheIssuesThatOccurredOnYourShift": {"blockedAmaAta": false, "contrabandFound": true, "medicationAbuse": true, "clientLeftAmaAta": true, "otherPleaseDescribe": false, "clientsNotFollowingRules": true, "clientsActingRomantically": false, "clientsUnhappyWithTreatment": false, "clientsNotParticipatingInTreatment": false, "clientsDisrespectfulToStaffOtherClients": false}, "didAnyMaintenanceissuesOccurOnYourShift": ""}
```

**Comment by wag110894** [comment]
@crivera305,

Thank you for reaching out! I am seeing the issue in the Jsfiddle that you provided, but am unable to copy the JSON that you provided above? Can you repost it so that we can investigate the form and try and locate the issue?

Thank you.

**Comment by crivera305** [comment]
try this i made a pastebin with just the form json - https://pastebin.com/8dkX2zqh

**Comment by wag110894** [comment]
@crivera305, 
Thanks for providing the form JSON. We have tested this on our side and are unable to replicate using the form JSON. Taking a look back at the JSfiddle, I am now unable to replicate any of the issues. Did you make any changes to the form?

**Comment by crivera305** [comment]
I have not made any changes. If there were anything that could have changed it could be a version update to formio.

The jsfiddle uses: https://unpkg.com/formiojs@latest/dist/formio.full.min.js which currently redirects to: https://unpkg.com/formiojs@4.11.0/dist/formio.full.min.js

After clearing cache the issue is still apparent in all browsers ( Firefox, Chrome & Safari ) - the panel loads with no internal components shown. Screenshot: https://ibb.co/mTmThKq

I have also created a video to demonstrate what I mean: https://streamable.com/tjti2h

- Page loads
- Scroll to area where Behavior Issues panel component loads
- Panel loads empty ( FAIL )

- I click "NO" for the Behavior Issues radio
- Behavior Issues panel loads with internal components
- Behavior Issues panel internal components values are blanked out ( FAIL )

** this demonstrates that subcomponents are available to the form on load - but are not displayed & javascript error is thrown

What i am expecting is that when the form loads - all internal components of a component should be shown.

There seems to be an error reflected in the video - screenshot of that for better clarity purposes: https://ibb.co/QHdbbCV

- Uncaught RangeError: Maximum call stack size exceeded
- [Violation] 'setTimeout' handler took 1915ms

Please advise. Thanks

**Comment by wag110894** [comment]
Hello @crivera305,

We took a further look at your code and see that the Radio for Behavior Issues Radio does not have the default value of Yes. This could be what is causing the problem. Can you add this setting and see if it is working?

**Comment by crivera305** [comment]
I believe you misunderstood what the issue is. Let me try to explain from top to bottom my experience and what the issue is.

The issue is not the default form ( that works as expected ).

In our case we do not want the form defaulted to YES - we want the user to click YES on the form to show the corresponding panel & questions. ( no reason to have everything showing  makes the form too long )

The issue is the form has already been filled out with the answer set to YES and when we render the formio form from JSON - the conditional handling is broken.

Example: 
- if i view a new form - all panels & hidden components are hidden - ( it functions as i expect it to )
- if i fill the form out and save it ( all answers are saved to JSON - https://pastebin.com/CigS4AUR )
     - notice `"didAnyBehavioralIssuesOccurOnYourShift": "yes",`
- if i view that JSON rendered on the form - formio does not work as expected.
     - The form shows the radio button for YES  selected, 
     - The Behavior Panel is displayed but all Internal components are missing !!! ( FAIL )

** please note no where in the logic do we show the panel without the internal components ** this is an error with Formio Rendering engine when rendering the form from the JSON i provided.


If you watch the video ( https://streamable.com/tjti2h ) you will notice that when Formio is rendering the form parsing the JSON ( https://pastebin.com/CigS4AUR ) - an error is thrown ( https://ibb.co/QHdbbCV )

This is all demonstrated in my first post which references my jsfiddle.  ( https://jsfiddle.net/0mkns9av/ )


When you run my fiddle you will notice the:
- form will load 
- form process the JSON ( https://pastebin.com/CigS4AUR )
- form ACKNOWLEDGES `"didAnyBehavioralIssuesOccurOnYourShift": "yes"` from JSON
     - shows the radio button "Behavioral issues" set to YES
     - shows the panel Behavior Panel
     - shows panel EMPTY ( FAIL )



Please review and let me know if i can clarify anything. Thanks for your assistance and time toward this matter.

**Comment by wag110894** [comment]
Hello @crivera305, Thank you for providing the additional information. We are able to replicate this and have created a ticket and have added it to our backlog. We welcome a pull request or have a process to expedite this issue if you are interested. 

If you are interested in expediting this issue, please contact support@form.io and reference this ticket number for this issue (FJS-1111).


We will let you know as soon as this issue has been resolved. 

Have a great weekend.

**Comment by jhen1422** [comment]
@crivera305 
This issue should be resolved in version 4.12.0-rc19 of formio.js.
